api/src/main/java/org/apache/iceberg/FileFormat.java

```diff
@@ -29,23 +29,33 @@ public enum FileFormat {
   PARQUET("parquet", true),
   AVRO("avro", true),
   // TODO(aduffy): Make Vortex splittable once I update FFI to allow providing split sizes.
-  VORTEX("vortex", false),
+  VORTEX("vortex", false, true),
   METADATA("metadata.json", false);
 
   private final String ext;
   private final boolean splittable;
+  private final boolean rowSplittable;
 
   private static final FileFormat[] VALUES = values();
 
   FileFormat(String ext, boolean splittable) {
+    this(ext, splittable, false);
+  }
+
+  FileFormat(String ext, boolean splittable, boolean rowSplittable) {
```

Comment:
@pvary This is contraversial, but it's the approach I took to allow Vortex to push split sizes down. Because Vortex does not use row groups, splits cannot be based on byte ranges but need to be in row ranges.

This gets wired up in the BaseContentScanTask